### DISCUSSION TITLE
asp-dotnet-core-in-action-third-edition-is-now-in-print/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#386] [Announcements]
# asp-dotnet-core-in-action-third-edition-is-now-in-print/

My new book, ASP.NET Core in Action, Third Edition, is now in print. In this post I describe some of the changes, how you can get it, and how to get money off

https://andrewlock.net/asp-dotnet-core-in-action-third-edition-is-now-in-print/

**Comment by macias** [comment]
On one hand it is great news (and congratulations to you), the bad thing is publisher. Manning has nasty habit of stamping (it is visible) **every** page with personal data (name + email) and on top of this using "of course" non-matching font. Maybe they read books in dark room, or something... ;-) After buying "Blazor in Action" and seeing how they damaged the book I said "stop". I know I am single outlier, so I don't count, but if more people are put off because of this, maybe consider changing publisher or publishing on your own in the future. Just thinking out loud. I hope some sane publisher will translate this in my native language so I could support your work at least this way.
